lms/djangoapps/certificates/models.py

```diff
@@ -86,6 +86,7 @@ class CertificateStatuses(object):
     regenerating = 'regenerating'
     restricted = 'restricted'
     unavailable = 'unavailable'
+    auditing = 'auditing'
```

Comment:
The docs at the top of this file should be updated, but I'm fine with that happening later in order to not delay the release.